test/commands/profiles/list.test.js

```diff
@@ -53,66 +56,154 @@ describe('commands', () => {
+            constants.FAKE_ACCOUNT_SID,
+            '',
+            constants.FAKE_API_KEY,
+            constants.FAKE_API_SECRET,
+          );
+          ctx.userConfig.addProject('profile2', constants.FAKE_ACCOUNT_SID);
+          ctx.userConfig.activeProfile = 'profile2';
+        })
+        .twilioCliEnv(Config)
+        .stdout()
+        .stderr()
+        .twilioCommand(ProfilesList, [])
+        .it('when the active profile is set from projects', (ctx) => {
+          expect(ctx.stdout).to.contain('profile1');
+          expect(ctx.stdout).to.contain('profile2');
+          expect(ctx.stdout).to.contain(constants.FAKE_ACCOUNT_SID);
+          expect(ctx.stdout).to.not.contain('Region');
+          expect(ctx.stdout.match(/true/g)).to.have.length(1);
+          expect(ctx.stdout).to.match(/profile2.*true/);
+          expect(ctx.stderr).to.equal('');
+        });
+
+      test
+        .do((ctx) => {
+          ctx.userConfig = new ConfigData();
+          ctx.userConfig.addProfile(
+            'default',
+            constants.FAKE_ACCOUNT_SID,
+            'dev',
+            constants.FAKE_API_KEY,
+            constants.FAKE_API_SECRET,
+          );
+          ctx.userConfig.addProfile(
+            'profile1',
+            constants.FAKE_ACCOUNT_SID,
+            '',
+            constants.FAKE_API_KEY,
+            constants.FAKE_API_SECRET,
+          );
+          ctx.userConfig.addProject('profile2', constants.FAKE_ACCOUNT_SID)
+        })
+        .twilioCliEnv(Config)
+        .stdout()
+        .stderr()
+        .twilioCommand(ProfilesList, [])
+        .it('runs profiles:list with 1 regional profile set and with multiple profiles', (ctx) => {
+          expect(ctx.stdout).to.contain('default');
+          expect(ctx.stdout).to.contain(constants.FAKE_ACCOUNT_SID);
+          expect(ctx.stdout).to.contain('dev');
+          expect(ctx.stdout).to.contain('Region');
+          expect(ctx.stdout.match(/true/g)).to.have.length(1);
+          expect(ctx.stderr).to.equal('');
+        });
+
+      test
+        .do((ctx) => {
+          ctx.userConfig = new ConfigData();
+          ctx.userConfig.addProfile(
+            'profile1',
+            constants.FAKE_ACCOUNT_SID,
+            '',
+            constants.FAKE_API_KEY,
+            constants.FAKE_API_SECRET,
+          );
+          ctx.userConfig.addProject('profile2', constants.FAKE_ACCOUNT_SID);
+        })
+        .twilioCliEnv(Config)
+        .stdout()
+        .stderr()
+        .twilioCommand(ProfilesList, [])
+        .it('runs profiles:list with multiple profiles in profiles and projects', (ctx) => {
+          expect(ctx.stdout).to.contain('profile1');
+          expect(ctx.stdout).to.contain('profile2');
+          expect(ctx.stdout).to.contain(constants.FAKE_ACCOUNT_SID);
+          expect(ctx.stdout).to.not.contain('Region');
+          expect(ctx.stdout.match(/true/g)).to.have.length(1);
+          expect(ctx.stderr).to.equal('');
+        });
```

Comment:
[Minor] Can remove this test, assertions seems to be covered with previous ones with mix of these, or could consider clubbing some of these assertions there

Reply:
Done.